.github/workflows/rspec-shared.yml

```diff
@@ -10,7 +10,10 @@ on:
         required: true
         type: string
       test-tag:
-        required: true
+        required: false
+        type: string
+      spec_paths:
```

Comment:
We're mixing `-` and `_`, and I just realized that `_` is more appropriate. Can you please change it to use `_` for all inputs in all workflows?

I know that's a bit out of scope for this PR, but we might as well fix it here.